bin/NativeTests/BigUIntTest.cpp

```diff
@@ -226,13 +226,12 @@ namespace BigUIntTest
     TEST_CASE("Init_From_Char_Of_Digits", "[BigUIntTest]")
     {
         BigUInt biDec;
-        char *charDigit;
```

Comment:
For what its worth, you could change this to just be `const char *charDigits` rather than redeclaring it in each block. `const char *charDigits` is ~equivalent to `char const *charDigits`, which makes it clearer when read right-to-left: "a pointer to a constant character". You would only have to redeclare charDigits in each block if it was `char const * const`, which is a constant pointer to a constant character (thus, you cant reassign the pointer). See https://godbolt.org/z/d3dAYe

Reply:
Great info! Thanks Jack!

---
In reply to: [216410613](https://github.com/Microsoft/ChakraCore/pull/5682#discussion_r216410613) [](ancestors = 216410613)